rook/doc/rook.tex

```diff
@@ -0,0 +1,81 @@
+\documentclass{article}
+
+\title{Rook Test Description}
+
+\begin{document}
+\maketitle
+
+\section{Testers}
+
+The testers are used to run a test.  The tester can fail the test, or
+it can be failed by one of the differs that are attached as subnodes
+in the test input.
+
+\subsection{Common Features}
+
+These are the features that are common to all testers.
+
+\begin{description}
+  \item[type] The type of this test
+  \item[skip] If true skip test
+  \item[prereq] list of tests to run before running this one
+  \item[max\_time] Maximum time that test is allowed to run in seconds
+  \item[os\_max\_time] Maximum time by os. Example: Linux 20 Windows 300 OpenVMS 1000
+  \item[heavy] If true, run only with heavy tests
+  \item[output] Output of the test
+  \item[expected\_fail] if true, then the test should fails, and if it passes, it fails
+  \item[run\_types] The run types that this test is
```

Comment:
Similar to above, I think it's more clear that `run_types` is like "heavy, qsub", but maybe listing that example would clarify at a glance which is used for which.

Reply:
Fixed.